src/Microsoft.DotNet.Interactive/Commands/Cancel.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.Interactive.Commands
+{
+    public class Cancel : KernelCommand
+    {
+        public Cancel(string targetKernelName = null): base(targetKernelName)
+        {
+            Handler = (command, context) => Task.CompletedTask;
```

Comment:
It's preferable to override `InvokeAsync`. We should at some point limit the settable `Handler` property to only those commands that vary behavior by kernel.